README.md

```diff
@@ -34,16 +34,13 @@ Unpack the `ct` binary, add it to your PATH, and you are good to go!
 A Docker image is available at `quay.io/helmpack/chart-testing` with list of
 available tags [here](https://quay.io/repository/helmpack/chart-testing?tab=tags).
 
-### From Source
+### Homebrew
```

Comment:
Glad for this change. People seem to keep getting confused at the from source options